Gulpfile.mjs

```diff
@@ -527,29 +527,37 @@ function buildRollup(packages, buildStandalone) {
 }
 
 function buildRollupDts(packages) {
-  const sourcemap = process.env.NODE_ENV === "production";
-  return Promise.all(
-    packages.map(async packageName => {
-      const input = `${mapToDts(packageName)}/src/index.d.ts`;
-      const output = `${packageName}/lib/index.d.ts`;
-      log(`Bundling '${chalk.cyan(output)}' with rollup ...`);
+  async function build(input, output, banner) {
+    log(`Bundling '${chalk.cyan(output)}' with rollup ...`);
 
-      const bundle = await rollup({
-        input,
-        plugins: [rollupDts()],
-        onwarn(warning, warn) {
-          if (warning.code !== "CIRCULAR_DEPENDENCY") warn(warning);
-        },
-      });
+    const bundle = await rollup({
+      input,
+      plugins: [rollupDts()],
+    });
 
-      await bundle.write({
-        file: output,
-        format: "es",
-        sourcemap: sourcemap,
-        exports: "named",
-      });
-    })
+    await bundle.write({
+      file: output,
+      format: "es",
+      banner,
+    });
+  }
+
+  const tasks = packages.map(async packageName => {
+    const input = `${mapToDts(packageName)}/src/index.d.ts`;
+    const output = `${packageName}/lib/index.d.ts`;
+
+    await build(input, output);
+  });
+
+  tasks.push(
+    build(
+      "packages/babel-parser/typings/babel-parser.source.d.ts",
+      "packages/babel-parser/typings/babel-parser.d.ts",
+      "/* eslint-disable import/no-extraneous-dependencies, @typescript-eslint/consistent-type-imports, prettier/prettier */"
```

Comment:
We can also ignore this file in our linter config as long as `source.d.ts` is properly linted.